src/sentry/models/processingissue.py

```diff
@@ -79,16 +80,65 @@ def find_resolved(self, project_id, limit=100):
         eventstore.bind_nodes(rv, "data")
         return rv, has_more
 
+    @staticmethod
+    def is_release_newer(project_id, release, other_release):
+        if other_release is None:
+            return True
+
+        try:
+            # TODO: we would need the organization to efficiently query the release.
+            release = Release.objects.get(projects__id=project_id, version=release)
+            other_release = Release.objects.get(projects__id=project_id, version=other_release)
```

Comment:
Ideally, query both of the releases with a single query. IIRC, this should be possible as:

```py
list(Release.objects.filter(projects__id=project_id, version__in=(release, other_release)))
```

Aside, it looks like there's no index on `(project_id, version)`, so it would be more efficient to query with the organization.

Reply:
Exactly, I was also thinking of the usage of `in`. I decided not as of now to not add complexity to match the version names but I can do that.